tools/tcprtt_example.txt

```diff
@@ -0,0 +1,25 @@
+Demonstrations of tcprtt, the Linux eBPF/bcc version.
+
+
+This program traces TCP RTT(round-trip time) to analyze the quality of
+network, then help us to distinguish the network latency trouble is from
+user process or physical network.
+
+# ./tcprtt.py -i 1 -d 10 -T
+Tracing TCP RTT... Hit Ctrl-C to end.
+
+18:29:05
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 0        |                                        |
+         8 -> 15         : 0        |                                        |
+        16 -> 31         : 0        |                                        |
+        32 -> 63         : 0        |                                        |
+        64 -> 127        : 3        |                                        |
+       128 -> 255        : 17       |                                        |
+       256 -> 511        : 23268    |*                                       |
+       512 -> 1023       : 707578   |****************************************|
+
+The above output shows that the TCP RTT is stable and the network seems good,
+if we hit a latency spike of a process, it maybe a problem of user code.
```

Comment:
1. Should add the USAGE message
Example: https://github.com/iovisor/bcc/blob/master/tools/funclatency_example.txt#L331

2. Should add more examples for the real-world problem.
For example, what kind of output means network problem instead of user code